src/EditorFeatures/Core/Implementation/GoToDefinition/GoToDefinitionHelpers.cs

```diff
@@ -20,9 +20,9 @@ internal static class GoToDefinitionHelpers
             ISymbol symbol,
             Project project,
             IEnumerable<Lazy<INavigableItemsPresenter>> presenters,
-            ITypeSymbol containingTypeSymbol,
-            bool throwOnHiddenDefinition,
-            CancellationToken cancellationToken)
+            CancellationToken cancellationToken,
+            bool thirdPartyNavigationAllowed = true,
+            bool throwOnHiddenDefinition = false)
```

Comment:
If there's an analyzer, it's not yelling at me. @Pilchie, what style did we decide on here? Since this is internal I don't want to make the cancellation token be optional.